src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerArgumentBinder.cs

```diff
@@ -205,107 +205,79 @@ public class ControllerArgumentBinder : IControllerActionArgumentBinder
             return modelBindingResult;
         }
 
-        // Called via reflection.
-        private static void CallPropertyAddRange<TElement>(object target, object source)
```

Comment:
Moving this method just makes the PR harder to review. I _think_ the method didn't change as you moved it. Please confirm.